storybook/index.html

```diff
@@ -145,5 +145,18 @@ <h1 class="sectionHeading">Hello</h1>
                 <p>Our program is constantly establishing new standards of excellence in the practice of public and contemporary art creation and exhibition. Our collaborative dialogue processes empower artists and communities to address and organize issues in the community. Community art builds bridges from conceptual connection to a physical understanding. With close attention to our clients desires the results are beautiful murals that inspire and motivate viewers.</p>
             </div>
         </section>
+
+        <section class="demo">
+            <div class="testimonial">
+                <div class="testimonial--pic">
+                    <img src="https://cdn.pixabay.com/photo/2015/04/23/22/00/tree-736885__340.jpg">
+                </div>
+                <div class="testimonial--content">
+                    <h1>Community Member</h1>
+                    <p>“Whether I shall turn out to be the hero of my own life, or whether that station will be held by anybody else, these pages must show.”</p>
+                </div>
+                <p>Our program is constantly establishing new standards of excellence in the practice of public and contemporary art creation and exhibition. Our collaborative dialogue processes empower artists and communities to address and organize issues in the community. Community art builds bridges from conceptual connection to a physical understanding. With close attention to our clients desires the results are beautiful murals that inspire and motivate viewers.</p>
```

Comment:
I think you have some of the mission statement left over...

<img width="626" alt="Screen Shot 2020-09-14 at 4 28 42 PM" src="https://user-images.githubusercontent.com/733916/93147899-5dafe000-f6a7-11ea-92cc-db9ff4729ce9.png">

Reply:
oh lol, on it